tests/comfy_cli/command/test_node_deps.py

```
@@ -314,6 +325,555 @@ def test_no_names_reports_every_pack(workspace, fake_pip):
+    cache["pack:someone-elses-expired"] = {"value": "1.0.0", "ts": time.time() - 999_999}
+    outdated_cmd._save_cache(cache)
+
+    node_deps_cmd.build_report(str(workspace), registry_ids=["some-pack"], python="/fake/python", registry_api=api)
+
+    saved = outdated_cmd._load_cache()
+    assert stale_key not in saved
+    assert fresh_key in saved
+    assert "pack:someone-elses-expired" in saved, "pruning is scoped to this command's own key space"
+
+
+def test_cache_round_trips_a_non_ascii_registry_id(workspace, fake_pip):
```

Pylint (4.0.6)

[warning] Redefining name 'workspace' from outer scope (line 87)

(W0621)